src/pages/Beneficiary.tsx

```diff
@@ -21,30 +23,37 @@ const Beneficiary = () => {
     (state: RootState) => state.bene.beneficiaryList
   );
 
+  const handleOpen = (openValue: boolean) => {
+    setOpen(openValue);
+    dispatch(clearBeneLoading({}));
+  };
+
   React.useEffect(() => {
```

Comment:
You can always just import useEffect on the top just to make it a little more clean if you want! 
import React, { useEffect } from 'react';